src/modules/trust/services/trust.service.ts

```
@@ -0,0 +1,232 @@
+import { getLogger } from '@logtape/logtape';
+import { trustTransactionsRepository } from '@/modules/trust/database/queries/trust-transactions.queries';
+import type { InsertTrustTransaction, SelectTrustTransaction } from '@/modules/trust/database/schema/trust-transactions.schema';
```

ESLint

[error] 'InsertTrustTransaction' is defined but never used. Allowed unused vars must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] `@/shared/database` import should occur before type import of `@/shared/types/result`

(import/order)